doc/news/changes.h

```diff
@@ -100,6 +100,11 @@ inconvenience this causes.
 
 
 <ol>
+ <li> New: Added tangent vector computation in FunctionManifold()
+ <br>
+ (Luca Heltai, 2016/03/18)
```

Comment:
Remove this for now -- the changelog will eventually gain an entry that talks about all of the changes we've made, but we haven't individually addressed any of them so far. See #2316.